src/registry.js

```diff
@@ -328,23 +329,38 @@ function getTraceType(traceType) {
  *  the dictionary mapping input strings to localized strings
  *  generally the keys should be the literal input strings, but
  *  if default translations are provided you can use any string as a key.
+ * @param {object} module.format
+ *  a `d3.locale` format specifier for this locale
+ *  any omitted keys we'll fall back on en-US
```

Comment:
Hmm. Isn't the fallback `en` ?

Reply:
to be precise, we'll fall back on no translation (which means `en-US`) - more detail https://github.com/plotly/plotly.js/pull/2207#discussion_r157288668